src/App.js

```diff
@@ -96,13 +101,32 @@ const App = ({ stateCode, correctionsData, jailsData }) => {
     releasesChartData.sourceData,
   ]);
 
+  const { jailsKeyInsightsData, coveredCounty } = generateJailsKeyInsightsData(jailsMetricData);
+
+  const incarcerationRateChartData = generateJailsChartData(
+    jailsMetricData,
+    INCARCERATION_RATE_JAIL,
+    ["Statewide", "US_CO_ARAPAHOE"],
+    ["Statewide", "Arapahoe county"],
+    coveredCounty
+  );
+
+  const incarcerationRateTopCountiesChartData = generateJailsChartData(
+    jailsMetricData,
+    INCARCERATION_RATE_JAIL,
+    ["US_CO_DENVER", "US_CO_EL_PASO", "US_CO_ARAPAHOE", "US_CO_JEFFERSON", "US_CO_ADAMS"],
+    ["Denver county", "El Paso county", "Arapahoe county", "Jefferson county", "Adams county"]
```

Comment:
Are you planning to make this dynamic in the current PR or in a follow-up PR, @nasaownsky?

Reply:
I'm planning to make this dynamic in this PR.